setup.py

```diff
@@ -50,6 +50,7 @@
     "tw2.forms",
     "WebHelpers2",
     "zope.sqlalchemy >= 0.4",
+    "Pillow",
```

Comment:
where is this used?

Reply:
`Pilloe` is needed to upload images. See this LOC: https://github.com/ColoradoSchoolOfMines/acm-website/blob/8729e9f163e2ac883949cb201e215c5d2c68e646/acmwebsite/model/banner.py#L12